src/components/dropdown/spec.coffee

```diff
@@ -106,24 +105,6 @@ export default () ->
       getHexagonElementDataObject(button.node()).eventEmitter.has('mouseover').should.equal(true)
       getHexagonElementDataObject(button.node()).eventEmitter.has('mouseout').should.equal(true)
 
-
-    it 'should set the alignment correctly', ->
-      dd = new Dropdown(button, content, {align: 'rbrb'})
-      dd._.alignments.should.eql('rbrb'.split(''))
-
-    it 'should use the right alignment option when a named align value is used', ->
-      dd = new Dropdown(button, content, {align: 'up'})
-      dd._.alignments.should.eql('ltlb'.split(''))
-
-      dd = new Dropdown(button, content, {align: 'down'})
-      dd._.alignments.should.eql('lblt'.split(''))
-
-      dd = new Dropdown(button, content, {align: 'left'})
-      dd._.alignments.should.eql('ltrt'.split(''))
-
-      dd = new Dropdown(button, content, {align: 'right'})
-      dd._.alignments.should.eql('rtlt'.split(''))
```

Comment:
`dd._.alignments` is no longer stored as we work it out using the option directly each time the dropdown is rendered